File: components_page/components/card/body.py
import dash_bootstrap_components as dbc
import dash_html_components as html

cards = html.Div(
    [
        dbc.Card(
            dbc.CardBody("This is some text within a card body"),
            className="mb-3",
        ),
        dbc.Card("This is also within a body", body=True),
    ]
)

File: components_page/components/card/body.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

cards <- htmlDiv(
  list(
    dbcCard(
      dbcCardBody("This is some text within a card body"),
      class_name = "mb-3"
    ),
    dbcCard("This is also within a body", body = TRUE)
  )
)
